### PR TITLE
Update Samples to reflect breaking Nuget Release for OSIsoft.OMFIngress

### PR DESCRIPTION
**Pull request by jizhu5548**
Recently we have released a new version of our public nuget package to be 5.0.0. I am updating the samples on behalf of the work item listed here: https://dev.azure.com/osieng/engineering/_sprints/taskboard/Parse/engineering/2021/Q3/2021.07.07?workitem=90897

### CODE REVIEW

**Comment by kelseyduffy** [comment]
@jizhu5548  can you also add a new line to the HISTORY.md file for version 1.1.11, today's date, and however you want to summarize the change; as well as the version number at the top of the README.md file to 1.1.11 as well.

**Comment by jizhu5548** [comment]
> @jizhu5548 can you also add a new line to the HISTORY.md file for version 1.1.11, today's date, and however you want to summarize the change; as well as the version number at the top of the README.md file to 1.1.11 as well.

done!

File: HISTORY.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Version History
+## 1.1.11 / 2021-06-09
```
Comment (dcbroad3) [inline HISTORY.md]: Should be one more line of whitespace between the first and second line.

**Review by dcbroad3 — APPROVED** [review]

File: HISTORY.md (hunk 1 above)
Comment (jizhu5548) [reply HISTORY.md]: Wow I missed the date as well

**Review by cbardini — APPROVED** [review]

**Pull request merged**